### PR TITLE
sandbox

### PR DESCRIPTION
**Pull request by abandonned**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: destinations/salesforce.md [hunk 1]
```diff
@@ -18,34 +18,42 @@ Before configuring your source and destination on the RudderStack, please verify
...
 {% endhint %}
 
 * Give a name to the destination and click on **Next**. You should then see the following screen:
 
 ![Salesforce Connection Settings](../.gitbook/assets/image%20%2841%29.png)
 
-* Please provide your Salesforce username and password here, along with the access token. Then, click on **Next**. Salesforce will be enabled as a destination in RudderStack.
+* Provide your Salesforce username and password here along with the access token.
+
+* If you are using a sandbox environment for integration, enable this. Then, click on **Next**. Salesforce will now be enabled as a destination in RudderStack.
```
Comment (ameypv-rudder) [inline destinations/salesforce.md:38]: "If you are using a sandbox environment for integration, enable this"

-- this is unclear. What needs to be enabled? Should we update the screenshot above to reflect this new setting/option? Please confirm.

Comment (abandonned) [reply destinations/salesforce.md:38]: Yes once the webapp PR is merged the feature will be visible. its a check box in the "other settings" part which can be enabled and disabled.
Also will update the screenshot.

Comment (ameypv-rudder) [reply destinations/salesforce.md:38]: Sounds good - I can take care of these things once the PR is merged, just wanted your confirmation. Thanks.

**Review by ameypv-rudder — APPROVED** [review]

File: destinations/salesforce.md (hunk 1 above)
Comment (abandonned) [reply destinations/salesforce.md]: I have already updated the doc. You can take a look

Comment (ameypv-rudder) [reply destinations/salesforce.md]: Looks good, thanks!

**Pull request merged**